mineral-ui/mineral-ui
# [website] Fix Typography page bugs

### Description


- Fix incorrect fontSize on examples
- Fix incorrect Heading fontWeights
- Fix incorrect table contents

### Motivation and context

Closes #534 

### Screenshots, videos, or demo, if appropriate


http://534-typography-page-fix--mineral-ui.netlify.com/typography/

### How to test


1. `cd mineral-ui && npm start`
2. Navigate to the Typography page
3. Confirm each example matches spec and that the example is styled as specified

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist




### How does this PR make you feel?

![Dancing skeleton](https://media.giphy.com/media/1zKfBxfGosHLLcMdcu/giphy.gif)
